node/tests/TestUtilities.ts

```diff
@@ -662,6 +662,11 @@ export async function transactionTest(
         ]),
     );
     args.push(2);
+    baseTransaction.geopos(key18, ["Palermo", "Catania"]);
+    args.push([
+        [13.36138933897018433, 38.11555639549629859],
+        [15.08726745843887329, 37.50266842333162032],
```

Comment:
Should we test boundary conditions?

Reply:
Can you get more details - what do you mean?
Keep in mind that transaction IT covers very basic cases, just to confirm that command works find in a transaction.

Reply:
Perhaps testing boundary conditions in addition to the ones added:

  [0, 90], 
  [0, -90], 
  [180, 0],
  [0, 0]

Reply:
It is already present in another test https://github.com/valkey-io/valkey-glide/blob/7c5a122fd6defdcc5db718c0f1f1ecbc14e27552/node/tests/SharedTests.ts#L4534